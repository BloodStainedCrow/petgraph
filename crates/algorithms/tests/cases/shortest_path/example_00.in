version https://git-lfs.github.com/spec/v1
oid sha256:0868d99aa063f973bc47eb5747fd8873936754f9ab8bf295eaaf930946bed156
size 51
