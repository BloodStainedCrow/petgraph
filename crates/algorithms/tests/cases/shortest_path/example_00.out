version https://git-lfs.github.com/spec/v1
oid sha256:92ef26703823a4c8b2cc311103c1d5f8e6118dd5bf83127307ce077d9f4b65f8
size 17
